content/docs/catalogue/assets.md

```diff
@@ -0,0 +1,139 @@
++++
+title = "Assets"
+description = "Viewing assets."
+date = 2022-03-19T18:20:00+00:00
+updated = 2022-03-19T18:20:00+00:00
+draft = false
+weight = 52
+sort_by = "weight"
+template = "docs/page.html"
+
+[extra]
+lead = "Managing your assets"
+toc = true
+top = false
++++
+
+Assets in Open mSupply is a digital register to manage the creation and maintenance of assets. It allows health departments to keep track of, manage and maintain their assets.
```

Comment:
```suggestion
Assets in Open mSupply is a digital register to manage the creation and maintenance of assets. 

The asset catalogue provides a list of base asset items, and is pre-populated from the [WHO PQS Catalogue](https://apps.who.int/immunization_standards/vaccine_quality/pqs_catalogue/) (sections E003 & E004). You can then create an asset for each piece of equipment at a site using the [Equipment](/docs/coldchain/equipment/) section, with these assets based on the items in this catalogue. Doing so will give you base information, such as the manufacturer & model of an asset without having to enter that manually.
```

Reply:
I've changed this to ```The asset catalogue provides a list of base asset items, and is pre-populated from the [WHO PQS Catalogue](https://apps.who.int/immunization_standards/vaccine_quality/pqs_catalogue/) (sections E003 & E004). You can then create an asset based on the items in this catalogue using the [Equipment](/docs/coldchain/equipment/) section. Doing so will give you base information, such as the manufacturer & model of an asset without having to enter that manually.```

as it felt a bit easier to digest for me, but let me know if you think it doesn't offer enough information